vividus-tests/src/main/resources/story/system/mobile_app/MobileAppStepsTests.story

```diff
@@ -280,6 +280,7 @@ Meta:
     @requirementId 2112
 When I tap on element located `accessibilityId(menuToggler)`
 When I tap on element located `xpath(<menuQrCodeXpath>)`
+When I wait until element located `xpath(<scrollViewXpath>)` disappears
```

Comment:
shouldn't we wait until `xpath(<menuQrCodeXpath>)` disappears?

Reply:
occasionally `xpath(<menuQrCodeXpath>)` is also element on the top of screenshoot.